core/src/main/java/org/eclipse/hono/util/CredentialsConstants.java

```diff
@@ -121,6 +125,14 @@ public final class CredentialsConstants extends RequestResponseApiConstants {
      * The name of the default hash function to use for hashed passwords if not set explicitly.
      */
     public static final String DEFAULT_HASH_FUNCTION             = HASH_FUNCTION_SHA256;
+    /**
+     * The name of the supported rsa algorithm.
+     */
+    public static final String RSA_ALG                            = "RSA";
+    /**
+     * The name of the supported elliptic curve algorithm.
+     */
+    public static final String EC_ALG                            = "EC";
```

Comment:
this is a good addition. Can you also please replace the usage of the literal strings in all other locations in the code? That shouldn't be too many occurrences ...

Reply:
sure, done